src/patterns/pages/workspace-dashboard/workspace-dashboard.template.html

```diff
@@ -1,147 +1,455 @@
+                  <div class="sam search">
+                    <input class="sam input" id="input-type-text" name="input-type-text" type="text">
+                    <button class="sam tertiary button">
+                      <span class="fa fa-search" aria-hidden="true"></span>
+                    </button>
+                  </div>
+                </div>
+          </sam-segment>
           </div>
-        </div>
-        <div class="event">
-          <div class="content">
-            <div class="summary">
-              <a class="action">Delete Opportunity</a> for A. Lewis
-              <div class="date">Monday</div>
-            </div>
-            <div class="meta">
-              <em>Details and name of organization</em>
-            </div>
+
+          <!-- FEDERAL HIERARCHY -->
+          <div class="fedassist">
+          <sam-segment>
+                <h3 class="sam heading middle aligned">
+                  <!-- SVG: Created by Henry Chung from the Noun Project -->
+                  <span class="icon">
+                    <img src="\assets\icon\federal hierarchy 1.svg" class="federal icon" alt="">
+                  </span>
+                  <a class="sam semibold link arrow">
+                    Federal Hierarchy
+                  </a>
+                </h3>
+
+                <div class="sam visible tablet up">
+                  <label class="sam input label" for="input-type-text">
+                    Federal Hierarchy organization lookup
+                  </label>
+                  <div class="sam search">
+                    <input class="sam input" id="input-type-text" name="input-type-text" type="text">
+                    <button class="sam tertiary button">
+                      <span class="fa fa-search" aria-hidden="true"></span>
+                    </button>
+                  </div>
+                </div>
+          </sam-segment>
           </div>
-        </div>
-        <div class="event">
-          <div class="content">
-            <div class="summary">
-              <a class="action">Assign Role</a> to J. Wong 
-              <div class="date">Nov 12</div>
+    </div>
+    <!-- ************************** 3 SAM SEGMENT ****************************-->
+    <div class="sam search cards">
+
+      <!-- USER DIRECTORY -->
+      <div class=" user search">
+        <sam-segment>
+          <h3 class="sam heading middle aligned">
+            <!-- SVG: Created by Henry Chung from the Noun Project -->
+            <span class="icon">
+              <svg xmlns="http://www.w3.org/2000/svg" xmlns:xlink="http://www.w3.org/1999/xlink" version="1.1" x="0px" y="0px" viewBox="0 0 100 125" style="enable-background:new 0 0 100 100;" xml:space="preserve">
+                <path d="M77.5,60C67.8,60,60,67.8,60,77.5C60,87.2,67.8,95,77.5,95S95,87.2,95,77.5C95,67.8,87.2,60,77.5,60z M77.5,91.9  c-7.9,0-14.4-6.5-14.4-14.4c0-7.9,6.5-14.4,14.4-14.4s14.4,6.5,14.4,14.4C91.9,85.4,85.4,91.9,77.5,91.9z"/>
```

Comment:
Now that we have all svg inside the assets folder - could you replace this with the svg path